src/crypto/aes_transit_crypto.ts

```diff
@@ -0,0 +1,61 @@
+import * as b64 from "base64-url"
+import * as crypto from "crypto"
+import {CryptoBase} from "./crypto_base"
+
+export class AESTransitCrypto extends CryptoBase {
+  ALGORITHM = "aes-256-ctr"
+
+  async encrypt(plaintext: string) {
```

Comment:
It's surprising we have such a complicated crypto implementation. Are we doing something special here or can we use something more off-the-shelf?